tests/tests_default.yml

```diff
@@ -6,8 +6,6 @@
   roles:
     - linux-system-roles.network
   tasks:
-    - name: Include the task 'el_repo_setup.yml'
```

Comment:
What is the reason for removing this task ?

Reply:
I put it back